components/Cards/ResourceCard/ResourceCard.css

```diff
@@ -0,0 +1,110 @@
+.ResourceCard {
+  width: 400px;
+  font-family: var(--secondaryFontFamily), sans-serif;
+  justify-content: normal;
+  box-shadow: 0 3px 6px 2px rgba(0, 0, 0, 0.12);
+  transition: 0.2s box-shadow linear;
+}
+
+.ResourceCard:focus,
+.ResourceCard:hover {
+  box-shadow: 0 3px 6px 2px var(--primary);
+}
+
+@media screen and (--small-viewport) {
+  .ResourceCard {
+    width: calc(100% - 5rem); /* subtracting margin/padding */
+  }
+}
+
+.titleSection {
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.titleSection img {
+  height: 50px;
+}
+
+.titleSection h5 {
+  margin-left: 0.5rem;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+@media screen and (--small-viewport) {
+  .titleSection h5 {
+    font-size: 1.25rem;
+  }
+}
+
+.descriptionText {
+  line-height: 1.25;
+  font-weight: 200;
+  overflow: hidden;
+  position: relative;
+  height: 6.2em; /* exactly five lines */
```

Comment:
I don't think it matters in this case since the card's font size is the same as the root's, but my thinking was that since the intent is to fix the height to 5 lines we actually do want it based off of the element's fontsize and not the root's.

Reply:
If we're tryna do hard-set sizes, I guess I'd prolly prefer pixels.

Reply:
@kylemh 
Setting the height with pixels hurts accessibility as it doesn't scale well with zoom.

With em's:
100%
![image](https://user-images.githubusercontent.com/27715246/62428146-c1559480-b6c3-11e9-82b2-03fac89d2f97.png)

200%
![image](https://user-images.githubusercontent.com/27715246/62428150-d29ea100-b6c3-11e9-9c92-01aaccba2174.png)

With pixels:
100%
![image](https://user-images.githubusercontent.com/27715246/62428165-1abdc380-b6c4-11e9-9e72-f90d801ae7bd.png)

200%
![image](https://user-images.githubusercontent.com/27715246/62428168-290bdf80-b6c4-11e9-918b-10898be655b9.png)

Reply:
I see the bad difference, but why would the container's dimensions would affect the way that the last line gets sliced? All text sizes are already set via `rem`s.